front/components/assistant/conversation/AssistantBrowerContainer.tsx

```diff
@@ -0,0 +1,78 @@
+import { Page } from "@dust-tt/sparkle";
+import type {
+  LightAgentConfigurationType,
+  WorkspaceType,
+} from "@dust-tt/types";
+import { useCallback } from "react";
+
+import { AssistantBrowser } from "@app/components/assistant/AssistantBrowser";
+import { useProgressiveAgentConfigurations } from "@app/lib/swr";
+import { classNames } from "@app/lib/utils";
+
+interface AssistantBrowserContainerProps {
+  onAgentConfigurationClick: (agent: LightAgentConfigurationType) => void;
+  owner: WorkspaceType;
+  setAssistantToMention: (agent: LightAgentConfigurationType) => void;
+}
+
+export function AssistantBrowserContainer({
```

Comment:
It could be used as we refactor the gallery. Additionally, even though it isn't clear in our hierarchy, there are distinct components. We have functional components like this one and visual components like the `AssistantBrowser`.

You'll find the same pattern on the input bar, it makes easier to maintain.